.env.example

```diff
@@ -11,7 +11,7 @@ DB_PASSWORD=secret
 
 CACHE_DRIVER=file
 SESSION_DRIVER=file
-QUEUE_DRIVER=sync
+QUEUE_DRIVER=database
```

Comment:
We should advocate a queue, but advise `sync` for people on cPanel etc... Imagine having hundreds or thousands of subscribers, it'd take ages to send all of the emails.